### PR TITLE
#161254617 reset password

### PR DESCRIPTION
**Pull request by Eyansky**
#### What does this PR do?
- Creates the front end interface for reset password.

#### Description of Task to be completed?
- A user should be presented with a button and or link to reset their passwords on the SignUp/Register component.

#### How should this be manually tested?
- git clone this repo
```
$ git clone https://github.com/andela/ah-orcas-frontend/
```
- git checkout to this branch
```
$ git checkout ft-reset-pass-161254617
```
- Install the dependencies
```
$ npm install
```
- Start the app
```
- npm start
```
- Check the button on the browser.

#### Any background context you want to provide?
- This only presents a working interface for the backend work. It will only bring a successful message to an email that has been signed up. 
- If your email is successful then you should also receive that email in your inbox.

#### What are the relevant pivotal tracker stories?
- [#161254617](https://www.pivotaltracker.com/story/show/161254617)

#### Screenshots (if appropriate)
screenshot 2018-12-11 at 16 48 29
screenshot 2018-12-11 at 16 48 06
screenshot 2018-12-11 at 16 47 54
screenshot 2018-12-11 at 16 47 38

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

**Review by TheSteelGuy — APPROVED** [review]
LGTM

File: src/components/ResetPassword/style.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  margin-bottom: 20em;
+  margin-left: 15.8em;
+  margin-top: 1em;
+  padding: 1em;
+}
+
+.form-content { 
+  margin-left: 15em;
+  margin-top: 2em;
+}
```
[the remark this answers is not in the document]
Comment (Eyansky) [reply src/components/ResetPassword/style.scss:21]: Already fixed

File: src/components/ResetPassword/style.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.my-container { 
+  margin-left: -3em;
+}
+
+.mybtn {
```
[the remark this answers is not in the document]
Comment (Eyansky) [reply src/components/ResetPassword/style.scss:27]: fixed

**Review by wrotich — APPROVED** [review]
LGTM

**Review by code-sleuth — APPROVED** [review]

**Pull request merged**